channels/ON-CHAIN.md

```diff
@@ -220,7 +223,7 @@ Serialization defined [here](../serializations.md#channel-close-solo-transaction
 Proof of inclusion represents a subset of the internal channel state. At the
 bare minimum it has to include all accounts and their balances but can also
 include contracts and contract calls. It must provide enough information to
-close the channel. Miners are to validate it and use its data to initiate the
+close the channel. Miners are to check balances in it if any and use this data to initiate the
```

Comment:
Not sure for the `if any` part because of the explicit expectations that all participant's balances must be present in the PoI

Reply:
The only case that comes to my mind is one party with zero balance, or with account not existing at all. Possible?

Reply:
Nope. Removing the `if any` part